arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,214 @@
+                    },
+                    {
+                      "name": "serverName",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "description": "The name of the Azure SQL Server."
+                    }, 
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_GetServerPolicy",
+                "description": "Returns an Azure SQL Server security alert policy.",
+                "parameters": [
+                    {
+                      "name": "resourceGroupName",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "description": "The name of the Resource Group to which the server belongs."
+                    },
+                    {
+                      "name": "serverName",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "description": "The name of the Azure SQL Server."
+                    }, 
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    }
+                }
+            }
+        }
+    },
+  "definitions": {
+    "ServerSecurityAlertPolicyProperties": {
+      "properties": {
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The state of the Azure SQL Security alert policy (New, Enabled or Disabled)."
```

Comment:
Do you want this to be an enum ? Take a look at tdocumentation of enum over [here] (https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#Enum-x-ms-enum)